python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -450,9 +450,10 @@
 )
 
 raise_on_partial_outputs_option = MWOptionDecorator(
-    "--raise-on-partial-outputs",
+    "--raise-on-partial-outputs/--no-raise-on-partial-outputs",
```

Comment:
That's cool, I didn't know you could do that.

Reply:
I learned about it on this ticket, too!